src/vm/dllimport.cpp

```diff
@@ -5747,25 +5755,34 @@ class LoadLibErrorTracker
                 priority = const_priorityCouldNotLoad;
                 break;
         }
-
-        UpdateHR(priority, HRESULT_FROM_WIN32(dwLastError));
+        UpdateHRWin(priority, HRESULT_FROM_WIN32(dwLastError));
+#endif
     }
 
     // Sets the error code to HRESULT as could not load DLL
     void TrackHR_CouldNotLoad(HRESULT hr)
     {
-        UpdateHR(const_priorityCouldNotLoad, hr);
+        UpdateHRWin(const_priorityCouldNotLoad, hr);
     }
     
     HRESULT GetHR()
     {
         return m_hr;
     }
 
+    SString GetMessage()
```

Comment:
This should return `SString&`.

Returning it as `SString` is quite inefficient - it creates copies of the whole string.